config.md

```diff
@@ -25,8 +25,7 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 **`root`** (object, REQUIRED) configures the container's root filesystem.
 
 * **`path`** (string, REQUIRED) Specifies the path to the root filesystem for the container.
-  The path can be an absolute path (starting with /) or a relative path (not starting with /), which is relative to the bundle.
-  For example (Linux), with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
+  The path can is either an absolute path or a relative path to the bundle. On Linux for example, with a bundle at `/to/bundle` and a root filesystem at `/to/bundle/rootfs`, the `path` value can be either `/to/bundle/rootfs` or `rootfs`.
```

Comment:
You're dropping the [one sentence per line][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc3/style.md#one-sentence-per-line